openpype/hosts/fusion/hooks/pre_pyside_install.py

```diff
@@ -0,0 +1,153 @@
+import os
+import subprocess
+import platform
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+
+
+class InstallPySideToFusion(PreLaunchHook):
+    """Automatically installs Qt binding to fusion's python packages.
+
+    Check if fusion has installed PySide2 and will try to install if not.
+
+    For pipeline implementation is required to have Qt binding installed in
+    fusion's python packages.
+    """
+
+    app_groups = {"fusion"}
+    order = 2
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+        # Prelaunch hook is not crucial
+        try:
+            settings = self.data["project_settings"][self.host_name]
+            if not settings["hooks"]["InstallPySideToFusion"]["enabled"]:
```

Comment:
This will crash in OpenPype all the time.

Reply:
Added Settings to pure Openpype, tested there too.